### ISSUE TITLE
Notification don't show the real connection

### ISSUE THREAD

**Issue by Damn3d** [gilbsgilbs/freemobilenetstat#27]
Hi.

I have a new phone since yesterday, Alcatel IDOL4S on Android 6.0.1

I'm very often in 4G Free but sometimes, i'm in 3G or 3G+ on Free / Orange. But the window notification of Free Mobile Netstat always show "Réseau mobile 4G (LTE)" and don't change with the real network connection.

No root, i've gave all the rights i can for this app. Nothing (for me) in Free Mobile Netstat option

Thank you for your return :)

**Comment by gilbsgilbs** [comment]
Hi, thanks for your report.

Is this a multi-SIM phone? If so, is the secondary SIM an Orange one? Could you try without this SIM and tell me if it works?

It might be a duplicate of issue #26 or issue #6.

Have a great day.
Gilbs

**Comment by Damn3d** [comment]
Yes it's a Multi-SIM phone but i have only one Free SIM (Nano-SIM) inside !

Sorry it it's a duplicate but in doubt, i've decide to register and signal the issues :)

**Comment by gilbsgilbs** [comment]
You did well :wink: . We really need as much reports as possible in order to know which phones are impacted by which bugs!
If you don't have another SIM inserted, I definitely think it's related to issue #6 (and maybe issues #26 and #6 are even related, hard to know) .

**Comment by Damn3d** [comment]
Humm maybe, if you say, you are the devs, not me :)

So you're on it for future update? ;)

**Comment by gilbsgilbs** [comment]
Unfortunately, there are no good fix I'm aware of (at least in FMN scope).
As far as I can tell, it seems to impact dual-SIM phones mostly and be
related to cell events that are not sent to the app.

Yet, on the One+2 I already implemented an ugly workaround that forces the
update periodically. I think the best we can do would be to generalise this
workaround to other phones such as yours.

I may ask you some further details about your phone in the future and
release a beta-test before I deploy the definitive workaround on the Play
Store. I'm not sure when I'll be able to do that (or when some contributer
will be able to do that).

**Comment by Damn3d** [comment]
Humm... i understand that any new android version or phone technology are a new challenge for improvement etc...

Don't hesitate, if you want something for information and/or if you deploy en beta .apk. I will very happy to help you because i really like your application. Indispensable when you have Free Mobile :)
